src/pymovements/transforms.py

```diff
@@ -213,7 +214,7 @@ def pos2vel(
     return v
 
 
-def vnorm(arr: np.ndarray, axis: int | None = None) -> np.ndarray:
+def vnorm(arr: np.ndarray, axis: int | None = None) -> np.ndarray | Any:
```

Comment:
is it Any because numpy doesn't fully support typehinting yet?

Reply:
yes -- they did not type linalg (output) and hence it is Any or float....